src/components/HowItWorksDialog.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <base-dialog
+    :id="`how-it-work-dialog`"
```

Comment:
I think it might make sense to make all the names the same, how-it-works throughout the file, considering the file name is how-it-works. I think it will cut down on confusion for someone else reading the file.